src/web/mjs/connectors/templates/WordPressMangastream.mjs

```diff
@@ -22,7 +24,7 @@ export default class WordPressEManga extends Connector {
         return data.map(element => {
             return {
                 id: this.getRootRelativeOrAbsoluteLink(element, request.url),
-                title: element.text.trim()
+                title: element.title.trim()
```

Comment:
Is it safe to assume all websites based on this template have a `title` attribute set on the manga link?

Reply:
No if i assume it worked OK before. I changed it to `title: element.title ? element.title.trim() : element.text.trim()` to be sure.